concrete/tools/conversations/add_message.php

```diff
@@ -90,22 +94,22 @@
     }
 }
 
-if (!Loader::helper('validation/token')->validate('add_conversation_message', $_POST['token'])) {
+if (!$app->make('helper/validation/token')->validate('add_conversation_message', $_POST['token'])) {
     $ve->add(t('Invalid conversation post token.'));
 }
 if (!$vs->notempty($_POST['cnvMessageBody'])) {
     $ve->add(t('Your message cannot be empty.'));
 }
 
 $parent = null;
-if (Loader::helper('validation/numbers')->integer($_POST['cnvMessageParentID']) && $_POST['cnvMessageParentID'] > 0) {
+if ($app->make('helper/validation/numbers')->integer($_POST['cnvMessageParentID']) && $_POST['cnvMessageParentID'] > 0) {
```

Comment:
This validator is being built several times in this file, I know it already was but it seems like an easy thing to fix